docs/architecture/incremental.md

```diff
@@ -0,0 +1,22 @@
+# Configuration: Streams & Incremental Syncs
+
+## AirbyteStream
+* The `AirbyteStream` has two sections:
+    * The `discovered` section describes information and metadata about the data in the source. This section is generated by the source connector. Outside of the source, this section should be treated as read only.
+    * The `configuration` section describes functionality that an end user would like applied on the sync. This section is configured by the end-user, and is never touched by the source.
+
+## Discovered
+
+Currently this section contains two fields: `json_schema` and `metadata`.
+* `json_schema` describes the structure of the data in a source.
+* `metadata` describes other information about the data in the source. For example what features are supported for the stream. Currently the `metadata` object contains two fields.
```

Comment:
This piece doesn't match the definition

Reply:
will remove docs for now. causing confusion as we iterate on the spec.